src/pages/ServiceDetails/ServiceInfo/ServiceInfoRouteRules/RouteRuleRoute.tsx

```diff
@@ -139,9 +165,21 @@ class RouteRuleRoute extends React.Component<RouteRuleRouteProps> {
   render() {
     return (
       <div style={{ marginTop: '30px' }}>
+        <div>
+          <BulletChart
+            id="bar-chart-1"
+            label="Load"
+            stacked={true}
+            thresholdWarning={100}
+            thresholdError={100}
+            details="Versions weight"
+            values={this.bulletChartValues()}
+            showLegend={true}
```

Comment:
Is the legend working? Should it be set as false?

Reply:
Yes, I was playing with this and I forgot remove it